src/devices/JkBms.cpp

```diff
@@ -95,6 +95,9 @@ bool JkBms_recvAnswer(uint8_t *p_lRecvBytes)
     if (mPortJk->available() > 0)
     {
       u8_lRecvByte = mPortJk->read();
+
+      ESP_LOGD(TAG,"%02x ",u8_lRecvByte);
```

Comment:
Entweder in `#ifdef JK_DEBUG` setzen oder entfernen.
Im laufenden Betrieb muss nicht jedes Byte ausgegeben werden.